subjects/devops/plus/README.md

```diff
@@ -0,0 +1,43 @@
+## plus
+
+### Instructions
+
+In this exercise, you will make a script that will take two arguments from the command line, add them and output the result.
+
+### Usage
+
+```console
+$ ./plus.sh 2 3 | cat -e
+5$
+$
+```
+
+### Hints
+
+Here are some commands that you can combine to find the solution:
+
+- `expr`: this command evaluates an expression and outputs the result.
+
+- `command substitution - $(...)`: this command can be used to interpolate the output of running a command into a string. You can replace the ellipsis with your command.
+
+You could use `expr` like this:
+
+```console
+$ expr 2 - 2
+0
+$
+```
+
+And `$(...)` like this:
+
+```console
+$ echo "This is the content of a file: $(cat file.txt)"
+This is the content of a file: <content of the file>`
+$
+```
+
+### Notions
```

Comment:
We are using this `### References` instead

Reply:
Thanks. Changed.